core/contracts/test/MockAdmin.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity ^0.5.0;
+
+import "../AdminInterface.sol";
+
+
+// A mock implementation of AdminInterface, taking the place of a financial contract.
+contract MockAdmin is AdminInterface {
```

Comment:
I forgot about this `Admin` naming scheme. I feel like it's super confusing (since the Admin is really the one calling it), but I can't think of anything better.... I almost want to call this an `AdministrateeInterface` jsut to annoy @madrasi.

Reply:
hahahahahha I love it, done.